tests/test_ostinato.py

```diff
@@ -78,3 +80,60 @@ def test_deterministic_ostinatoed(seed, dask_cluster):
         npt.assert_almost_equal(ref_radius, comp_radius)
         npt.assert_almost_equal(ref_Ts_idx, comp_Ts_idx)
         npt.assert_almost_equal(ref_subseq_idx, comp_subseq_idx)
+
+
+@pytest.mark.parametrize(
+    "seed", np.random.choice(np.arange(10000), size=25, replace=False)
+)
+def test_random_ostinato_with_isconstant(seed):
+    isconstant_custom_func = functools.partial(
+        naive.isconstant_func_stddev_threshold, quantile_threshold=0.05
+    )
+
+    m = 50
+    np.random.seed(seed)
+    Ts = [np.random.rand(n) for n in [64, 128, 256]]
+    Ts_subseq_isconstant = [isconstant_custom_func for _ in range(len(Ts))]
+
+    ref_radius, ref_Ts_idx, ref_subseq_idx = naive.ostinato(
+        Ts, m, Ts_subseq_isconstant=Ts_subseq_isconstant
+    )
+    comp_radius, comp_Ts_idx, comp_subseq_idx = stumpy.ostinato(
+        Ts, m, Ts_subseq_isconstant=Ts_subseq_isconstant
+    )
+
+    npt.assert_almost_equal(ref_radius, comp_radius)
+    npt.assert_almost_equal(ref_Ts_idx, comp_Ts_idx)
+    npt.assert_almost_equal(ref_subseq_idx, comp_subseq_idx)
+
+
+@pytest.mark.parametrize("seed", [79, 109, 112, 133, 151, 161, 251, 275, 309, 355])
+def test_deterministic_ostinatoed_with_isconstant(seed, dask_cluster):
+    isconstant_custom_func = functools.partial(
+        naive.isconstant_func_stddev_threshold, quantile_threshold=0.05
+    )
+
+    with Client(dask_cluster) as dask_client:
+        m = 50
+        np.random.seed(seed)
+        Ts = [np.random.rand(n) for n in [64, 128, 256]]
+
+        l = 64 - m + 1
+        subseq_isconsant = np.full(l, 0, dtype=bool)
+        subseq_isconsant[np.random.randint(0, l)] = True
+        Ts_subseq_isconstant = [
+            subseq_isconsant,
+            None,
+            isconstant_custom_func,
+        ]
```

Comment:
Please see the lines above. Is it okay that the list `Ts_subseq_isconstant` contains different types in the unit testing? My goal is:
(1) to consider different types for T_subseq_isconstant
(2) avoid having constant subseq in one of the time series (in `Ts`) as I think the radius beocmes 0 otherwise. That is why one of element in `Ts_subseq_isconstant` is `None`.

Reply:
I think this is okay.